telegram_bot/bot_handlers/handle_view_user_achievements.py

```diff
@@ -0,0 +1,26 @@
+from database.users_quiz_progress_table import fetch_user_achievements
+from telegram_bot.bot_message_generators.statistic_menu_generator import send_user_statistic_menu
+
+
+async def handle_view_user_achievements(call, bot):
+    chat_id = call.message.chat.id
```

Comment:
You can immediately pass this value to send_user_statistic_menu, just like username for example